README.md

```diff
@@ -1,4 +1,106 @@
 # hacktoberbot
-A community project for Hacktoberfest 2018. A Discord bot primarily designed to help teach Python learners from the PythonDiscord community how to contribute to open source.
+A community project for [Hacktoberfest 2018](https://hacktoberfest.digitalocean.com). A Discord bot primarily designed to help teach Python learners from the PythonDiscord community how to contribute to open source.
 
 You can find our community by going to https://discord.gg/python
+
+## Motivations
+We know it can be difficult to get into the whole open source thing at first. To help out, we've decided to start a little community project during hacktober that you can all choose to contribute to if you're finding the event a little overwhelming, or if you're new to this whole thing and just want someone to hold your hand at first.
+
+*Lemon*
+
+## Commands
+
+### repository
+Alias: project
+A command to send the hacktoberbot github project
+
+### git
+Alias: github
+A command that sends a getting started with Git page
+
+### git.commit
+Sends a sample first commit
+
+## Getting started
+
+If you are new to this you will find it far easier using PyCharm:
```

Comment:
It feels a bit like this readme completely mandates PyCharm. it would be great if in addition to a "with pycharm" approach, there was a "without PyCharm" approach as well... but I realize this may be a big ask. What are your thoughts?

Reply:
orginally I planned to do this however I have no idea how to setup things like environment variables in Windows (I can in Arch but not sure how helpful that is)